### ISSUE TITLE
Audio not working

### ISSUE THREAD

**Issue by ccapontep** [fminetto/unofficial-deezer-player#4]
The app on Ubuntu 16.04 shows the song playing but no audio is hear neither through the laptop speaker nor when connecting headphones. Deezer works fine when opening it in the web browser.

**Comment by veljkho** [comment]
> The app on Ubuntu 16.04 shows the song playing but no audio is hear neither through the laptop speaker nor when connecting headphones. Deezer works fine when opening it in the web browser.

Hey @ccapontep, it should be due to permissions. Just go to Settings > Applications > Unofficial Deezer.. and tick "Play and record sound". Then just reopen Deezer app completely and it will play the sound.
Note for @fminetto please note this somewhere in the app description on the store so people know about this, I also had this problem and thought the app is not working but actually tricky problem.

**Comment by ccapontep** [comment]
Thanks @veljkho. The version of Ubuntu 16.04 does not have an 'Application' option in Settings. But I was able to resolve by going to Ubuntu Software and opening the Deezer App and clicking on Permissions (maybe it should prompt this question when downloading). I attached a picture showing it.

![image](https://user-images.githubusercontent.com/34521579/82695690-9547d780-9c33-11ea-93a8-de854e337c66.png)
